src/plugins/i18n.ts

```diff
@@ -0,0 +1,28 @@
+// Copyright 2019 Daniel Erat and Niniane Wang. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file. 
+
+import Vue from 'vue';
+import VueI18n from 'vue-i18n';
+
+Vue.use(VueI18n);
+
+const messages = {
+  'en': {
+    yourName: 'Your Name',
+    individual: 'Individual',
+    team: 'Team',
+  }, 
+  'es': {
+    yourName: 'Tu Nombre',
+    individual: 'Individual',
+    team: 'Grupo',
+  }
+};
+
+export const i18n = new VueI18n({
```

Comment:
if this is going to be the only export, should we just do something like `export default new VueI18n(...`?

Reply:
(i might be getting the syntax wrong)

Reply:
Ok, I used export default.